include/grpcpp/impl/codegen/rpc_service_method.h

```diff
@@ -41,13 +41,18 @@ class MethodHandler {
   virtual ~MethodHandler() {}
   struct HandlerParameter {
     HandlerParameter(Call* c, ServerContext* context, void* req,
-                     Status req_status)
-        : call(c), server_context(context), request(req), status(req_status) {}
+                     Status req_status, std::function<void()> renew)
+        : call(c),
+          server_context(context),
+          request(req),
+          status(req_status),
+          renewer(std::move(renew)) {}
     ~HandlerParameter() {}
     Call* call;
     ServerContext* server_context;
     void* request;
     Status status;
+    std::function<void()> renewer;
```

Comment:
what does this do?

Reply:
It issues a new request after this one finishes.

Reply:
I added this comment in the documentation for the constructor.